package.json

```diff
@@ -106,5 +105,29 @@
     "test": "snyk test",
     "postinstall": "node scripts/npm/apps-install"
   },
-  "snyk": true
+  "snyk": true,
+  "babel": {
```

Comment:
lets move the babel and jest stuff to config files.

Reply:
Moved to `.babelrc` and `jest.config.json` in https://github.com/cfpb/cfgov-refresh/pull/3862/commits/d54399bb5e5b426c7a93b92d75ea6b4392939a3a. I wanted to put the jest config in /test/ but had trouble with the paths resolving for the coverage source files.